packager/react-packager/src/node-haste/DependencyGraph/ResolutionRequest.js

```diff
@@ -385,7 +381,22 @@ class ResolutionRequest {
             );
           });
 
-          return p;
+          return p.catch(error => {
+            if (error.type !== 'UnableToResolveError') {
+              throw error;
+            }
+            throw new UnableToResolveError(
+              fromModule,
+              toModuleName,
+              `Module does not exist in the module map ${searchQueue.length ? 'or in these directories:' : ''}\n` +
```

Comment:
@davidaurelio Actually I use `path.dirname` with `searchQueue.map` to ensure that each line is the parent directory. So "in these directories" makes perfect sense. 😄